ostd/src/arch/x86/trap/trap.S

```diff
@@ -57,6 +57,7 @@ trap_handler_table:
 .section .text
 .global trap_common
 trap_common:
+    cld                     # clear DF flag set by previous states to ensure consistent string ops
```

Comment:
I think the fundamental reason of clearing this flag is to satisfy the requirement of the calling convention (see _System V Application Binary Interface AMD64 Architecture Processor Supplement_). So I suggest revising the comment to reflect this point.

```
    cld                    # DF must be clear on function entry according to x86-64 calling convention
```